snafu/uperf_wrapper/uperf_wrapper.py

```diff
@@ -54,6 +54,13 @@ def __init__(self, parent_parser):
         self.args.num_pairs = os.getenv("num_pairs", "")
         self.args.multus_client = os.getenv("multus_client", "")
         self.args.networkpolicy = os.getenv("networkpolicy", "")
+        self.args.nodes_in_iter = os.getenv("node_count", "0")
+        self.args.pod_density = os.getenv("pod_count", "0")
+        self.args.colocate = os.getenv("colocate","False")
+        self.args.step_size = os.getenv("stepsize","")
+        self.args.density_range = os.getenv("density_range","0-0")
```

Comment:
Can you add a comment on what this 0-0 means?

Reply:
@mrbojangles3 , this is a list kind of field from the CR
node_range: [2,4] for example. I'm of the opinion we keep this as a list/array/ here as well, not sure about the "0-0" kind of string notation.

Reply:
Also, to restate if we dont have the field default should be empty.